src/apps/backend/modules/access-token/rest-api/access-token-serializer.ts

```diff
@@ -0,0 +1,7 @@
+import { AccessToken } from '../types';
+
+export const serializeAccessToken = (accessToken: AccessToken): unknown => ({
```

Comment:
Recommend to add `AsJSON` here and other places as suffix.

In an ideal world, based on request params (json, csv, xml, etc), we may have different serializers. Adding suffix makes it bit more clear and future proof.

Reply:
Addressed